### PR TITLE
React Todo App: Added todo items CRUD operations and SPA routing features

### PR DESCRIPTION
**Pull request by gtekle**
## What this PR does:
  - Added React project setup with GitHub Actions, ESlint, and stylelint
  - Added home page to add, update, delete and display todo items
  - Added todo item completed status update feature
  - Added about page to explore nested react-router routing mechanism
  - Added persistence in Local Storage feature of todo list
  - Added react svg icons for buttons
  - Added hamburger menu to show/hide side nav

## To test it
  - clone repository from `https://github.com/gtekle/react-todo-app.git`
  - run `cd react-todo-app` to switch to project root directory
  - run `npm install`
  - run `npm start`
 
## Details

### CODE REVIEW

File: src/components/InputTodo.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        onChange={onChange}
+        required
+      />
+      <button type="button" className="input-submit">
+        <FaPlusCircle
+          style={{
+            color: 'darkcyan',
+            fontSize: '20px',
+            marginTop: '2px',
+          }}
```
Comment (mrparkersson) [inline src/components/InputTodo.js:45]: -When the` FaPlusCircle` is clicked, an event is not fired to add a todo. 
I think implementing a function to this button will make your project nicer and complet
Screen Shot 2022-02-15 at 8 21 33 AM

**Review by mrparkersson — CHANGES_REQUESTED** [review]
Hi @gtekle  👋🏻,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Github Flow.
✅ Good PR description

Required Changes ♻️

Check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/InputTodo.js (hunk 1 above)
Comment (gtekle) [reply src/components/InputTodo.js:45]: Hi @mrparkersson
Thank you very much for the feedback. I will address each and every one of them.
thanks again.

**Review by mrparkersson — APPROVED** [review]
Hi @gtekle 👋🏻,

Good job so far!
Your project is complete and its time to merge 💪 
_Cheers and Happy Coding_

To Highlight 👏🏻

✅ No linter errors.
✅ You used correct Github Flow.
✅ Good PR description

Required Changes ♻️

Check the comments under the review.

Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by gtekle** [comment]
Hi @mrparkersson 
thanks!

**Pull request merged**